examples/Client.py

```diff
@@ -10,7 +10,7 @@ def main():
 
     parser = ArgumentParser(main.__doc__)
     parser.add_argument('--torrent', type=str, help='Path of the Torrent file', required=True)
-    parser.add_argument('--peers', type=str, help='Path to file contain peers (in the format ip:port for each line)')
+    parser.add_argument('--peers', type=str, help='Path to file contain peers (in the format ip:port for each line) or string in the same format but delimited by commas')
```

Comment:
Ok you just use  ```os.path.isfile``` nice

Reply:
The only problem is if the user has a file named like the way the strings are splitted. If that's the case the program will read the file instead of setting the peers. But it would be really weird for someone to have a file with such name.

Reply:
yeah that would be super edge case and the user fault